services/bettersocial/post/createPostV3.js

```diff
@@ -330,7 +330,7 @@ const BetterSocialCreatePostV3 = async (req, isAnonimous = true) => {
     else post = await Getstream.feed.createPost(req?.token, data);
 
     // Upvote post before sending the success message
-    await upVote(post?.id, req.token, req?.userId);
+    await upVote(post?.id, req.token, userDetail?.user_id);
     countProcess(post?.id, {upvote_count: +1}, {upvote_count: 1});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `countProcess` function call seems to be unchanged, but ensure that it can handle the potential undefined value of `post?.id` gracefully. If not, consider adding error handling or a null check for `post`.

```diff
-    countProcess(post?.id, {upvote_count: +1}, {upvote_count: 1});
+    if (post && post.id) {
+        countProcess(post.id, {upvote_count: +1}, {upvote_count: 1});
+    } else {
+        // Handle error or fallback scenario here
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->